.github/workflows/runtimes-matrix.json

```diff
@@ -41,7 +41,6 @@
     "uri": "wss://kusama-asset-hub-rpc.polkadot.io:443",
     "is_relay": false,
     "blocktime": 6000,
-    "extra_args": "--disable-mbm-checks",
```

Comment:
Can you try with blocktime 12000 ? It is still on 12s, not sure if this upgrade maybe enables async backing with 6 seconds, but it can cause the try-runtime error from the CI.